content/en/account_management/org_settings/cross_org_visibility.md

```diff
@@ -57,6 +57,16 @@ Creating a cross-organization connection allows you to query metrics from the so
 1. In the drop-down menu, select the _destination_ organization where you want to see the data.
 1. Click **Connect**.
 
+### Update a connection
+
+Updating a cross-organization connection allows you to change the data types that are shared from the source organization in the destination organization for an existing connection.
+
+1. Make sure you are signed in to the _source_ organization that contains the data you want to expose.
```

Comment:
Just want to double check if `the data you want to expose` is the phrasing we want here? If the update were to consist of removing data types, would that still be accurate?